### PR TITLE
i18n: rest of the de translations for dt

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts [hunk 1]
```diff
@@ -12,11 +12,92 @@ const triggerSet: TriggerSet<Data> = {
   triggers: [],
   timelineReplace: [
     {
-      locale: 'en',
-      replaceText: {
+      'locale': 'en',
+      'replaceText': {
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:16]: Good catch, I totally forgot that we quote properties in the `timelineReplace` object when I wrote this.

File: ui/raidboss/data/07-dt/trial/zoraal-ja.ts [hunk 2]
```diff
@@ -531,82 +540,122 @@ const triggerSet: TriggerSet<Data> = {
   timelineReplace: [
     {
       'locale': 'de',
-      'missingTranslations': true,
       'replaceSync': {
         'Fang': 'Reißzahn',
+        'Shadow of Tural': 'Schatten Turals',
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/trial/zoraal-ja.ts:545]: Here and elsewhere, in both this and the FRU translation block, as with the UCoB PR you just did, can you capitalize "of", "an", "the", etc? I know some of these lines are pre-existing from previous PRs, but it should be okay to update those alongside your new additions.

**Review by JLGarber — CHANGES_REQUESTED** [review]

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts (hunk 1 above)
Comment (Akurosia) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:16]: this is just pure luck that my script did this xD

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts [hunk 3]
```diff
@@ -12,11 +12,92 @@ const triggerSet: TriggerSet<Data> = {
...
+        'Blastburn': 'Brandstoß',
+        'Blasting Zone': 'Erda-Detonation',
+        'Burn Mark': 'Brandmal',
+        'Burnished Glory': 'Leuchtende Aureole',
+        'Burnout': 'Brandentladung',
+        'Burnt Strike': 'Brandschlag',
+        'Cyclonic Break': 'Zyklon-Brecher',
+        'Explosion': 'Explosion',
+        'Fall of Faith': 'Sünden-Erdspaltung',
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:35]:
```suggestion
        'Fall Of Faith': 'Sünden-Erdspaltung',
```

I'm so sorry, could I also get you to run the capitalization on "of" and "of the" for the FRU replacements as well?

Comment (Akurosia) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:35]: Should i also turn "Turn of the Heavens" into "Turn Of The Heavens" (in FRU)?

Comment (Akurosia) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:35]: Same with "Dawn of an Age" in Zoraal Ja

Comment (JLGarber) [reply ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts]: Yes please.

**Review by JLGarber — APPROVED** [review]

**Pull request merged**